### DISCUSSION TITLE
WIKI.js not listed in search engines only serves the purposes of Google

### DISCUSSION THREAD

**Discussion by darashayda** [requarks/wiki#6060] [General]
I found out that your organization intentionally not allowing the listing the WIKI pages here in Google and other search engines. 

This move, at best is idealogical and only serves the purpose of Google and make them more powerful. 

What we have against Google and Apple and others is their mighty power change the policies , block this and allow that kill this and promote that! 

We do not want them to have that power. 

The overseers here are exercising the same absolute power over our WIKI pages! 

They should leave it up to individuals to decide index their WIKI pages or not! 

Ideologically driven decisions never bear fruits 

Dara

**Comment by darashayda** [comment]
No one should read these comments as attacks against them or wiki.js, on the contrary I hope we are here for long time to come. 

But more than often genius people suffer from some vision issues which require some discourse.

**Comment by netthier** [comment]
Can you elaborate on what exactly you're talking about? The official homepage and docs show up on Google:
<img src="https://user-images.githubusercontent.com/66856670/212935531-a2535766-d8f8-4b6d-a9ce-7072342a3b73.png" width="500">
And while self-hosted instances have a robots.txt disallowing all access by default (which makes a ton of sense because not all wikis are supposed to be public), there is an option for disabling that: 
<img src="https://user-images.githubusercontent.com/66856670/212936083-2abdd757-361d-40ff-96e5-c385a239d80a.png" width="500">

**Comment by darashayda** [comment]
Wiki’s do, not my home page

http://wiki.ccnstudios.com <http://wiki.ccnstudios.com/>   

Nothing of my site shows up anywhere. 

But Wiki.js’ pages are all indexed. 

However something at the installation on Digital Ocean blocks communications with search engines (some process I am not  an expert at). 

Unless I made a mistake, Wiki.js is serving the purposes of Google and the rest of giants. 

Worst, Wiki.js is as much a iron fisted organization as is Google, but adding policies to customers who need their wiki pages indexed and available on the web. 

I think some old school thinking has clogged this otherwise great documentation project, and my IT director is very unhappy with my  choice for Wiki.js on this account!

 Please provide a second-hand approach to manually list our pages on Google!

I cannot use Google Search Console, since the site above belongs to wik.js domain registration and I have no authority to force a search/index on my site 

D
